src/components/overlays/AppMarketplaceRequest/index.tsx

```diff
@@ -77,20 +79,21 @@ export default function AppMarketplaceRequest({ id }: { id: string }) {
   ]
 
   const handleConfirmApp = async (id: string) => {
-    try {
-      const data = agreements?.map((agreement) => {
-        return {
-          agreementId: agreement.agreementId,
-          consentStatusId:
-            checkedAgreementsIds.indexOf(agreement.agreementId) >= 0
-              ? 'ACTIVE'
-              : 'INACTIVE',
-        }
-      })
-      data && addSubscribeApp({ appId: id, body: data }).unwrap()
-    } catch (err) {
-      console.log('error', err)
-    }
+    const data = agreements?.map((agreement) => {
+      return {
+        agreementId: agreement.agreementId,
+        consentStatusId:
+          checkedAgreementsIds.indexOf(agreement.agreementId) >= 0
```

Comment:
instead of

`list.indexOf(item) >= 0` 

better use more readable:

`list.includes(item)`